const reporter = store => next => action => {
  try {
    return next(action);
  } catch (err) {
    console.error('Caught !', err);
    Raven.captureException(err, {
      extra: {
        action,
        state: store.getState()
      }
    });
    throw err;
  }
};
